articles/TOC.yml

```diff
@@ -1,33 +1,54 @@
-- name: Welcome
+- name: Quantum Development Kit Preview Documentation
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
-- name: Quantum computing concepts
-  topicHref: concepts/index.md
-  href: concepts/toc.yml
-- name: Quickstart - your first quantum program
-  href: quickstart.md
-- name: Managing quantum machines and drivers
-  topicHref: machines/index.md
-  href: machines/toc.yml
-- name: Quantum development techniques
-  topicHref: techniques/index.md
-  href: techniques/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Q# programming language
-  topicHref: language/index.md
-  href: language/toc.yml
-- name: Contributing to the Quantum Development Kit
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: Glossary
-  href: Glossary.md
-- name: For more information
-  href: for-more-info.md
-- name: Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
-- name: License
-  href: license.md
+- name: Overview
+  items:
+  - name: Getting started
+    href: install-guide/index.md
+  - name: Contributing to the Quantum Development Kit
+    href: contributing/toc.yml
+    topicHref: contributing/index.md
+- name: Quickstarts
+  items:
+  - name: C#
+    href: install-guide/csharp.md
+  - name: Create a Bell State in Q#
+    href: quickstart.md
+  - name: Jupyter Notebooks
+    href: install-guide/jupyter.md
+  - name: Python
+    href: install-guide/python.md
+- name: Concepts
+  items:
+  - name: Quantum computing concepts
+    topicHref: concepts/index.md
+    href: concepts/toc.yml
+- name: How-to guides
+  items:
+  - name: Managing quantum machines and drivers
+    topicHref: machines/index.md
+    href: machines/toc.yml
+  - name: Quantum development techniques
+    topicHref: techniques/index.md
+    href: techniques/toc.yml
+- name: Reference
+  items:
+  - name: Glossary
+    href: Glossary.md
+  - name: Q# libraries
+    href: libraries/toc.yml
+  - name: Q# programming language
+    topicHref: language/index.md
+    href: language/toc.yml
```

Comment:
The Q#, .NET, and Python APIs are all also reference items, should they be linked to from here?

Reply:
The Q# Library Reference and C# Components Reference are currently linked from the TOC.yml file as they exist outside of docs.microsoft.com. I also would like to move them under References and need to discuss how to move them more with Carolyn and Natalie.  I'm not sure how they are referenced for the TOC at the moment.  Good call in noticing this.  I didn't see any Python specific library docs for the Quantum SDK. Is there a link you can share?  Thanks.